src/encoded/genomic_data_service.py

```diff
@@ -12,7 +12,13 @@
     'assayType': {'title': 'Assay (RNA SubType)'},
     'libraryPrepProtocol': {'title': 'Experiment'},
     'expressionID': {'title': 'File'},
```

Comment:
These should probably be snake case eventually to be consistent with other fields.